src/main/kotlin/org/koitharu/kotatsu/parsers/site/en/MyComicList.kt

```diff
@@ -0,0 +1,155 @@
+package org.koitharu.kotatsu.parsers.site.en
+
+import androidx.collection.arraySetOf
+import org.koitharu.kotatsu.parsers.MangaLoaderContext
+import org.koitharu.kotatsu.parsers.MangaSourceParser
+import org.koitharu.kotatsu.parsers.PagedMangaParser
+import org.koitharu.kotatsu.parsers.config.ConfigKey
+import org.koitharu.kotatsu.parsers.network.UserAgents
+import org.koitharu.kotatsu.parsers.model.*
+import org.koitharu.kotatsu.parsers.util.*
+import java.text.SimpleDateFormat
+import java.util.*
+
+@MangaSourceParser("MYCOMICLIST", "MyComicList", "en", ContentType.COMICS)
+internal class MyComicList(context: MangaLoaderContext) : PagedMangaParser(context, MangaParserSource.MYCOMICLIST, 24) {
+
+    override val configKeyDomain = ConfigKey.Domain("mycomiclist.org")
+    
+    override fun onCreateConfig(keys: MutableCollection<ConfigKey<*>>) {
+        super.onCreateConfig(keys)
+        keys.add(userAgentKey)
+    }
+
+    override val availableSortOrders: Set<SortOrder> = EnumSet.of(
+        SortOrder.UPDATED,
+        SortOrder.POPULARITY,
+        SortOrder.NEWEST,
+        SortOrder.ALPHABETICAL
+    )
+
+    override val filterCapabilities: MangaListFilterCapabilities
+        get() = MangaListFilterCapabilities(
+            isMultipleTagsSupported = false,
+            isSearchSupported = true,
+            isSearchWithFiltersSupported = false,
```

Comment:
```suggestion
            isSearchSupported = true,
```

`false` options can be omitted